lib/internal/crypto/util.js

```diff
@@ -25,7 +26,10 @@ const {
 var defaultEncoding = 'buffer';
 
 function setDefaultEncoding(val) {
-  defaultEncoding = val;
+  const enc = normalizeEncoding(val);
+  if (enc === undefined)
+    throw new errors.TypeError('ERR_UNKNOWN_ENCODING', val);
+  defaultEncoding = enc;
```

Comment:
this change does not seem to be directly related to the deprecation

Reply:
This should probably go into a separate PR if still wanted.

Reply:
Ah, right, yeah. missed this.